design-at-cornell/src/components/HomepageStyles.ts

```diff
@@ -131,4 +222,87 @@ export const Grid = styled.div`
   height: fit-content;
   margin-top: 90px;
   justify-content: center;
+  @media (max-width: 1000px) {
+    display: flex;
+    flex-direction: row;
+    height: fit-content;
+    justify-content: left;
+    margin-top: 0px;
+  }
+`;
+
+export const MobileElementContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-weight: normal;
+  margin-bottom: 50px;
+
+  p {
+    font-size: 24px;
+    text-align: left;
+    align-items: left;
+    width: 300px;
+    padding-top: 25px;
+  }
+
+  img {
+    cursor: pointer;
+  }
+`;
+
+export const ListsEvents = styled.div`
+  display: flex;
+  align-items: left;
+  margin-top: 100px;
+  margin-left: 40px;
+  margin-right: 40%;
+  flex-direction: column;
+  justify-content: left;
+  height: 290px;
+`;
+
+export const ListsArticles = styled.div`
+  display: flex;
+  align-items: left;
+  margin-top: 100px;
+  margin-left: 40px;
+  margin-right: 40%;
+  flex-direction: column;
+  justify-content: left;
+  height: 620px;
+`;
+
+export const SmallTitle = styled.div`
+  font-weight: bold;
+  text-align: left;
+  color: black;
+  align-items: left;
+  text-decoration: underline;
+  display: flex;
+  width: 60%;
+  box-sizing: border-box;
+  margin-top: 27px;
+  position: absolute;
+  font-size: 18px;
+  line-height: 23px;
+  margin-left: 50px;
+  cursor: pointer;
+`;
+
+export const SmallHeading = styled.div`
```

Comment:
Similar to the comment about MobileElementContainer, for SmallHeading you can use export const SmallHeading = styled(SmallTitle) to reuse some styles.